index.js

```diff
@@ -359,6 +355,18 @@ instance.prototype.actions = function() {
 	}
 
 	self.setActions({
+		'StartRecording': {
+			label: 'Start Recording',
+		},
+		'StopRecording': {
+			label: 'Stop Recording',
+		},
+		'StartStreaming': {
+			label: 'Start Streaming',
+		},
+		'StopStreaming': {
+			label: 'Stop Streaming',
+		},
```

Comment:
To make these consistent with the rest of the code base, these actions should be in the format of `start_recording` instead of `StartRecording`

Reply:
I will change this in the next commit.

Reply:
Made a new pull request with only the right changes but something strange is happening when i save this project. Those lines at 84 and 155 gets removed even if I dont touch them.